### PR TITLE
feat: Add ID method to auth

### PR DESCRIPTION
**Pull request by milwad-dev**
## 📑 Description

```php
// Laravel
Auth::id();
```
```go
facades.Auth(ctx).Id()
```

@coderabbitai summary

## ✅ Checks

- [x] Added test cases for my code

### CODE REVIEW

File: auth/auth.go [hunk 1]
```diff
@@ -76,6 +77,29 @@ func (a *Auth) User(user any) error {
 	return nil
 }
 
+func (a *Auth) Id() (int, error) {
```
Comment (kkumar-gcc) [inline auth/auth.go:80]: Id may not be `int` type.

Comment (milwad-dev) [reply auth/auth.go]: I know. I just wrote a simple method, and I will complete it.

**Comment by milwad-dev** [comment]
All tests passed, but I don't know why Mockery failed!

**Comment by devhaozi** [comment]
> All tests passed, but I don't know why Mockery failed!

Because no permission, you can run mockery localy and commit changes.

**Comment by hwbrzzl** [comment]
Great PR, thanks! Added you to the contributor list.

**Comment by milwad-dev** [comment]
Thanks

**Comment by hwbrzzl** [comment]
Hey @YlanzinhoY  We created a private channel in Discord for contributors. You can click [the link](https://discord.gg/dMs73mSa) and DM me (@Bowen) to join it if you are interested.

Here you can:

1. Send PR link or review others' PRs;
2. Discuss new features before creating an issue;
3. Vote for something;
4. etc.

**Pull request merged**